bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -336,6 +331,7 @@ spec:
             --sb-client-privkey /ovn-cert/tls.key \
             --sb-client-cert /ovn-cert/tls.crt \
             --sb-client-cacert /ovn-ca/ca-bundle.crt
+            --config-file=/run/ovnkube-config/ovnkube.conf \
```

Comment:
@pecameron stray `\` at the end of the last line here, that needs to move up to the end of the previous line?

Reply:
Yup